### DISCUSSION TITLE
http access on current cover

### DISCUSSION THREAD

**Discussion by payback007** [jcorporation/myMPD#318] [Q&A] [answered]
Hi,

i've got an additional question: is it possible to get direct access to the current album / stream cover via a http access? The idea is after "embedding of myMPD" is now working perfect, I would also need access to the cover of the actuel running music on a different site out myMPD application.

**Comment by jcorporation** [comment, marked as answer]
The coverimage is always accessible over this url:
`http://jukebox/albumart/<musicfile relative to music_directory>`

**Comment by payback007** [comment]
Yes I found that already, but is there any posibility to have something like a call? Means I know the actuel path, but if there is a new file played, the path is also changing. Is there a request I can send to get back the right path to the albumart?

**Comment by jcorporation** [comment]
Curl in combination with jq does the trick for you, a simple shell script would look like:
```
URI=$(curl -s -k https://jukebox/api -d'{"jsonrpc":"2.0","id":0,"method":"MPD_API_PLAYER_CURRENT_SONG","params":{}}' | jq -r .result.uri)
echo "https://jukebox/albumart/${URI}"
```
The API documentation is your friend: https://github.com/jcorporation/myMPD/wiki/API

**Comment by payback007** [comment]
Sorry, I'm absolutely not familiar with json-rpc requests. Thank you, script is working in priciple. Is it also possible to get the output with a HTTP-GET request?

**Comment by jcorporation** [comment]
No, the API is POST only
